staff/monica-morales/precourse/bingo-pro/Index.js

```diff
@@ -0,0 +1,228 @@
+var bingoCard=[
+    { number: ''},
+    { number: ''},
+    { number: ''},
+    { number: ''},
+    { number: ''},
+    //next line
+    { number: ''},
+    { number: ''},
+    { number: ''},
+    { number: ''},
+    { number: ''},
+    //next line
+    { number: ''},
+    { number: ''},
+    { number: ''},
+    { number: ''},
+    { number: ''}
+];
+puntuacion=[
+    {turnos:20,puntos:10},
+    {turnos:30,puntos:9},
+    {turnos:40,puntos:8},
+    {turnos:50,puntos:7},
+    {turnos:60,puntos:6},
+    {turnos:70,puntos:5},
+    {turnos:80,puntos:4},
+    {turnos:90,puntos:3},
+];
+
+//contadores
+var aciertosLine1=0;
+var aciertosLine2=0;
+var aciertosLine3=0;
+var linea=0;
+var turnos=0;
+
+//crear carton con numeros aleatorios que no se repitan
+function carton(){
+    var numeros_carton=[]
+    for(i=0; numeros_carton.length<15;i++){
+        var numero=Math.floor(Math.random() * 99) + 1;
+        if(numeros_carton.indexOf(numero)===-1)
+        numeros_carton.push(numero);
+    }
+    for(i=0;i<15;i++){
+        bingoCard.splice(i,1,numeros_carton[i]);
+    }
+};
+
+//Función para mostrar el carton por lineas
+function print(){
+    var print1='';
+    var print2='';
+    var print3='';
+    for(i=0; i<bingoCard.length;i++){
+        if(i<5){
+            if(i===4){
+                print1=print1+bingoCard[i]; 
+            }else{
+                print1=print1+bingoCard[i]+'-';
+            }
+        }else if(i<10){
+            if(i===9){
+                print2=print2+bingoCard[i]; 
+            }else{
+                print2=print2+bingoCard[i]+'-';
+            }
+        }else if(i<=15){
+            if(i===14){
+                print3=print3+bingoCard[i];   
+            }else{
+                print3=print3+bingoCard[i]+'-';
+            }
+        }   
+    }
+   return `\n${print1}\n${print2}\n${print3}\n`;
+   
+};
+
+//crear numero aleatorio que no se repita. 'la bola del juego'
+var numeros_random=[]
+function generateRandom(){
+    var aleatorio=Math.floor(Math.random() * 99) + 1;
```

Comment:
I suggest Math.ceil(Math.random() * 99);  instead Math.floor(Math.random() * 99) + 1

You could delete + 1 to the final. And exchange ceil by floor.

Reply:
Thanks! change done.